portal/views/assessment_engine.py

```diff
@@ -1314,6 +1344,119 @@ def assessment_add(patient_id):
             questionnaire:
               display: EPIC 26 Short Form
               reference: https://stg.us.truenth.org/api/questionnaires/epic26
+      - schema:
```

Comment:
Out of scope for this PR, but how would you feel about a static YAML (or JSON) file/directory to keep our schemas in (at least for the FHIR schemas)? Would make our view code more readable

Reply:
not opposed - i'd like to clean up the code by removing these massive comment blocks.  small fear they won't be as well maintained, but that's the life of a comment.  validation will always help with that.